lib/ansible/modules/files/unarchive.py

```diff
@@ -574,8 +579,21 @@ def unarchive(self):
         if self.excludes:
             cmd.extend(['-x'] + self.excludes)
         cmd.extend(['-d', self.dest])
-        rc, out, err = self.module.run_command(cmd)
-        return dict(cmd=cmd, rc=rc, out=out, err=err)
+        if self.password:
+            try:
+                import pexpect
+            except ImportError:
+                self.module.fail_json(msg="The pexpect python module is required for password option")
```

Comment:
This `try/except` should be moved to top level imports, and restructured to look like:

```
try:
    import pexpect
    HAS_PEXPECT = True
except ImportError:
    HAS_PEXPECT = False
```

And then in this block, use `if not HAS_PEXPECT:`